### PR TITLE
Feature/bl UI 3279 mobile view tabs

### PR DESCRIPTION
**Pull request by andrupu**
Fixes # BLUI-3279

#### Changes proposed in this Pull Request:

-Update dev docs styling to match the designs for mobile view on smaller screens.

-Main point of address will be updates to the Playground screen on AppBar Example.

-This will NOT include multilayer drawer updates.

#### To Test: 

- yarn start:reactdev

- https://www.figma.com/file/kiuLsThIDMqGLXMaL9cv33/Doc-It-Redesign?type=design&node-id=531-88905&mode=design&t=NCEdUTbbU3b5DkT0-0

### CODE REVIEW

**Review by surajeaton — COMMENTED** [review]
@andrupu  Prettier and lint are failing.
Please run yarn precommit.

**Comment by ektaghag-eaton** [comment]
@andrupu , can you add figma link ?

**Comment by daileytj** [comment]
> @andrupu , can you add figma link ?

https://www.figma.com/file/Nydz8n8HBj6U3ZtULja3gD/%F0%9F%92%AB-Brightlayer-UI-Tokens-%2B-Components-(in-Material-Design-3)?type=design&node-id=410%3A825&mode=design&t=gUHmaUsG82YTNfoB-1

File: docs/src/shared/PreviewComponentWithCode.tsx [hunk 1]
```diff
@@ -21,7 +26,7 @@ const PreviewComponentWithCode: React.FC<PreviewComponentProps> = (props): JSX.E
                      * We take the height of the appbar/tabs out of the 70vh portion
```
Comment (huayunh) [inline docs/src/shared/PreviewComponentWithCode.tsx:26]: Remember to update the inline comment

File: docs/src/shared/PlaygroundDrawer.tsx [hunk 2]
```diff
@@ -253,30 +274,135 @@ const PlaygroundDrawer = (props: DrawerProps): JSX.Element => {
         );
     };
 
+    type PlaygroundTabPanelProps = {
```
Comment (huayunh) [inline docs/src/shared/PlaygroundDrawer.tsx:277]: we usually take the type declaration out of the component scope.

Comment (huayunh) [reply docs/src/shared/PlaygroundDrawer.tsx:277]: In  fact, both `PlaygroundTabPanelProps` and `PlaygroundTabPanel` can probably be taken out of `PlaygroundDrawer` to make `PlaygroundDrawer` code shorter and easier to read

File: docs/src/shared/PlaygroundDrawer.tsx [hunk 3]
```diff
@@ -253,30 +274,135 @@ const PlaygroundDrawer = (props: DrawerProps): JSX.Element => {
...
-                            '& .MuiInputBase-root, & .MuiFormControlLabel-label': {
-                                fontFamily: '"Roboto Mono", monspace',
-                            },
-                            zIndex: theme.zIndex.appBar - 1,
-                            backgroundColor: 'background.paper',
+            <Drawer
+                sx={{
+                    '& .MuiPaper-root, & .MuiDrawer-root': {
+                        transform: 'none !important',
+                        visibility: 'visible !important',
```
Comment (huayunh) [inline docs/src/shared/PlaygroundDrawer.tsx:309]: You should not be turning the drawer on/off via these CSS attributes

**Review by huayunh — CHANGES_REQUESTED** [review]
- Please resolve your lint error about "Missing return type on function"
- Please run prettier
- Please add the black overlay thing (scrim) we mentioned. Ideally, onclick the black overlay should dismay the expanded bottom sheet
- Please add the animation to expand / collapse the drawer
- Please remove all the unused code that you commented out
- Something is causing an undesired overflow. For example, this should not be scrolling up and down like this:
https://github.com/etn-ccis/blui-react-component-library/assets/8997218/acf7693e-61b9-443e-a0e1-0610b37a4dae

**Comment by joebochill** [comment]
Closing due to inactivity. It's going to be a while before this refactoring work is ready to go into dev...I would suggest setting up a new feature branch to use as the target for the incremental work until it is ready to be merged into dev.

**Pull request closed without merging**